### ISSUE TITLE
Searching not triggered by typing title name

### ISSUE THREAD

**Issue by nixternal** [keepassxreboot/keepassxc#10377]
## Overview
Prior to 2.7.7 which I think I upgraded today via KDE Neon packages, when I would start typing a title name it would automatically start searching my database. That isn't working any more, I have to do Ctrl+F to bring up the search box first.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Open your database
2. Start typing the title of an account in your DB, ie Google

## Expected Behavior
If I start typing `google` when the database is unlocked & opened, KeepassXC will automatically search and show me the results, in this case I should see my Google account in the right hand pane to copy my password.

## Actual Behavior
Nothing happens right now. It did work prior to 2.7.7.

## Context
KeePassXC - 2.7.7

```
$ dpkg -l keepassxc
||/ Name           Version            Architecture Description
+++-==============-==================-============-========================================
ii   keepassxc      2.7.7-1ppa1~jammy1 amd64        KeePass Cross-Platform Community Edition
```
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Neon
Description:    KDE neon 6.0
Release:        22.04
Codename:       jammy
```

Operating System: Linux (KDE Neon)
Desktop Env: KDE
Windowing System: X11

**Comment by droidmonkey** [comment]
We didn't change anything regarding search behavior, it works perfectly fine for me on Windows, macOS, and Linux (Fedora KDE/Gnome/XFCE). This is also a well tested feature that our CI tests every single code change.

**Comment by nixternal** [comment]
Ok. Feel free to close this then and I'll wipe my config tomorrow and see.
If I still get it I'll go to the package. Appreciate your really quick
response.

**Comment by droidmonkey** [comment]
Come back if you can narrow a cause, sometimes package maintainers apply patches to our code. We do recommend using Flatpak instead (about to update it now).

**Comment by nixternal** [comment]
OK, this is strange. I removed the package and installed the Flatpak. Still the same thing. Just checked my apt.log to see if anything installed yesterday as well but nothing has installed since the 8th and between then & the 2.7.7 update the typing trigger was working. If I figure anything out, I will update you. Thanks again!

**Comment by nixternal** [comment]
Hey @droidmonkey it seems it is related to KDE Neon 6.x upgrade that happened recently. I suspect that when I did the upgrade, the system automatically switched to Wayland which didn't work well with others obviously. I switched back to X11 but I am guessing there is some sort of bug-a-boo somewhere in my system that has made KeePass mad. If I ever figure something out I'll pass it on. Thanks again!
